_posts/2022-12-01-Stumbling-Into-Greatness.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Stumbling-Into-Greatness"
+date: 2023-03-273T10:00:00

```

Comment:
@arceacereza This PR is failing tests because the date is incorrect.

Can you change this from `2023-03-273` to `2023-03-24`?

Reply:
Thanks @p-galligan! Just fixed the date.

Reply:
No, it should be fine.